package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cmd-extra",
-  "version": "1.0.0",
+  "version": "1.2.0",
```

Comment:
Probably no need to do this

Reply:
yes pls don't du bumps yet. we will do it in master during release.